GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -180,6 +180,23 @@ - (void)restorePreviousSignInWithCallback:(nullable GIDSignInCallback)callback {
   [self signInWithOptions:[GIDSignInInternalOptions silentOptionsWithCallback:callback]];
 }
 
+- (BOOL)restorePreviousSignInNoRefresh {
```

Comment:
To align the behavior with `restorePreviousSignInWithCallback`, we should immediately return YES and do nothing else if currentUser != nil.

Reply:
Done.